contents/blog/posthog-vs-ga4.md

```diff
@@ -0,0 +1,536 @@
+        <td className="text-center"><span className="text-green text-lg">✔</span></td>
+      </tr>
+      <tr>
+        <td>
+          <p class="!mb-0 pb-0.5 !leading-tight"><strong>Retention</strong></p>
+          <p class="!mb-0 !text-sm text-opacity-75 leading-none">Analyze user and revenue retention</p>
+        </td>
+        <td className="text-center"><span className="text-green text-lg">✔</span></td>
+        <td className="text-center"><span className="text-green text-lg">✔</span></td>
+      </tr>
+    </tbody>
+  </table>
+</div>
+
+PostHog also offers the following features to help you understand how people are using your product:
+
+<div className="overflow-x-auto -mx-5 px-5 w-screen md:w-auto">
+  <table className="w-full mt-4">
+    <thead>
+      <tr>
+        <td className="w-3/12"></td>
+        <td class="text-center"><strong>GA4</strong></td>
+        <td class="text-center"><strong>PostHog</strong></td>
+        <td></td>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>
+          <p class="!mb-0 pb-0.5 !leading-tight"><strong>Correlation analysis</strong></p>
+          <p class="!mb-0 !text-sm text-opacity-75 leading-none">Automatically highlight factors affecting funnel conversion rates</p>
+        </td>
+        <td className="text-center"><span className="text-red text-lg">✖</span></td>
+        <td className="text-center"><span className="text-green text-lg">✔</span></td>
+        <td></td>
+      </tr>
+      <tr>
+        <td>
+          <p class="!mb-0 pb-0.5 !leading-tight"><strong>Lifecycle insights</strong></p>
+          <p class="!mb-0 !text-sm text-opacity-75 leading-none">Break down events from new, returning, resurrecting, and dormant users</p>
+        </td>
+        <td className="text-center"><span className="text-red text-lg">✖</span></td>
+        <td className="text-center"><span className="text-green text-lg">✔</span></td>
+        <td></td>
+      </tr>
+      <tr>
+        <td>
+          <p class="!mb-0 pb-0.5 !leading-tight"><strong>Stickiness insights</strong></p>
+          <p class="!mb-0 !text-sm text-opacity-75 leading-none">See how many times users perform an event in a period of time</p>
+        </td>
+        <td className="text-center"><span className="text-red text-lg">✖</span></td>
+        <td className="text-center"><span className="text-green text-lg">✔</span></td>
+        <td></td>
+      </tr>
+      <tr>
+        <td>
+          <p class="!mb-0 pb-0.5 !leading-tight"><strong>Formulas</strong></p>
+          <p class="!mb-0 !text-sm text-opacity-75 leading-none">Use custom formulas to create unique insights</p>
+        </td>
+        <td className="text-center"><span className="text-red text-lg">✖</span></td>
+        <td className="text-center"><span className="text-green text-lg">✔</span></td>
+        <td></td>
+      </tr>
+      <tr>
+        <td>
+          <p class="!mb-0 pb-0.5 !leading-tight"><strong>SQL access</strong></p>
```

Comment:
Maybe a note that GA lets you query with SQL if you export to BigQuery, but in PostHog it is built in?